examples/rest-custom-ajv/package.json

```diff
@@ -17,8 +17,8 @@
 		"debug": "^4.1.1",
 		"express": "4.16.4",
 		"lodash": "4.17.21",
-		"mongoose": "^5.11.15",
-		"swagger-ui-dist": "^3.35.1"
+		"mongoose": "^5.13.14",
+		"swagger-ui-dist": "^4.1.3"
```

Comment:
it's worth checking if the swagger ui is working after the upgrade, you can do it by visiting the `/explorer` path of a running example

Reply:
hey Piero, I tested all the examples and the swagger is displaying correctly. I also tested the APIs a bit and seems to be responding fine.